### PR TITLE
Oleksii/ob 172 fizz buzz

### PR DESCRIPTION
**Pull request by alexone55**

### CODE REVIEW

File: Alexey/Text/fizz_buzz/main/fizz_buzz.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+def fizzbuzz_printer(number):
+    if number % 3 == 0 and number % 5 == 0:
+        return 'FizzBuzz'
+    elif number % 3 == 0:
+        return 'Fizz'
+    elif number % 5 == 0:
+        return 'Buzz'
+    else:
+        return number
```
Comment (glock3) [inline Alexey/Text/fizz_buzz/main/fizz_buzz.py]: your function should return single type
str or int. Please change it

**Review by glock3 — CHANGES_REQUESTED** [review]

File: Alexey/Text/fizz_buzz/main/fizz_buzz.py (hunk 1 above)
Comment (alexone55) [reply Alexey/Text/fizz_buzz/main/fizz_buzz.py]: Done

File: Alexey/Text/fizz_buzz/main/fizz_buzz.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+def fizzbuzz_printer(number):
+    if number % 3 == 0 and number % 5 == 0:
+        return 'FizzBuzz'
+    elif number % 3 == 0:
+        return 'Fizz'
+    elif number % 5 == 0:
+        return 'Buzz'
+    else:
+        return str(number)
```
Comment (glock3) [inline Alexey/Text/fizz_buzz/main/fizz_buzz.py]: Still there is no contract in the function. Should return 'Fizz', 'Buzz' or 'FizBuzz'.

File: Alexey/Text/fizz_buzz/main/fizz_buzz.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+def main():
+    numbers = int(input())
+    print(fizzbuzz_cycle(numbers))
+
+
+def fizzbuzz_cycle(numbers):
+    fizz_buzz_numbers = []
+    for number in range(1, numbers + 1):
```
Comment (glock3) [inline Alexey/Text/fizz_buzz/main/fizz_buzz.py:8]: Remove for loop. Make it more simple

**Review by glock3 — CHANGES_REQUESTED** [review]

**Review by glock3 — APPROVED** [review]

**Pull request merged**